test/stream.jl

```diff
@@ -69,5 +69,18 @@
 
         @test i == Int(elements ./ batch)
     end
-end
 
+    @testset "FilterModifier" begin
+
+        df = DataFrame(x = [1, 2, 3, 4, 5, 6], y = [6, 5, 4, 3, 2, 1], z = [6, 5, 4, 3, 2, 1])
+        conn_df = EasyStream.TablesConnector(df);
+        stream = EasyStream.BatchStream(conn_df; batch = 2);
+        filter = EasyStream.FilterModifier([:x, :x, :y])
+        push!(stream, filter)
+
+        stream_filtered = EasyStream.listen(stream)
+
+        @test propertynames(stream_filtered) == [:x, :y]
```

Comment:
Aqui seria um in. O symbol até faz sentido comparar, mas teria que garantir a lógica de comparação do array.

Reply:
não entendi o in. Seria algo como `propertynames(stream_filtered) in [:x, :y]` ?

Reply:
ou seria algo como 
```
:x in propertynames(stream_filtered)
:y in propertynames(stream_filtered)
```

Reply:
> ou seria algo como
> 
> ```
> :x in propertynames(stream_filtered)
> :y in propertynames(stream_filtered)
> ```

Acho que é deste jeito.